src/utilities/fe_dofs.jl

```diff
@@ -0,0 +1,66 @@
+type FEField
+    name::AbstractString # Descriptive string
+    dim::Int # Number of dimensions for the field
+end
+
+type FEDofs
```

Comment:
Okay yea, then you could only use the FEDofs object then, and only use the mesh object to set up the FEDofs?